inc/functions.php

```diff
@@ -6358,92 +6357,100 @@ function get_inactive_forums()
  * @param bool $fatal (Optional) Stop execution if it finds an error with the login. Default is True
  * @return bool|int Number of logins when success, false if failed.
  */
-function login_attempt_check($fatal = true)
+function login_attempt_check($uid = 0, $fatal = true)
```

Comment:
I doubt many plugins use this function, but we should make a note that the function signature has changed in the release notes.